files/katex/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
```

Comment:
Or change to github. I'm not sure what the `bower` file structure looks like. I haven't used KaTex myself so I don't know what files should be expected